main.tf

```diff
@@ -0,0 +1,65 @@
+provider "aws" {
+  region = "eu-central-1"
+}
+
+resource "aws_s3_bucket" "bucket" {
+  bucket = "cesko-digital-assets"
+  acl    = "public-read"
+}
+
+locals {
+  s3_origin_id = "s3-origin"
+}
+
+resource "aws_cloudfront_origin_access_identity" "origin_access_identity" {
+
+}
+
+resource "aws_cloudfront_distribution" "s3_distribution" {
+  origin {
+    domain_name = aws_s3_bucket.bucket.bucket_regional_domain_name
+    origin_id   = local.s3_origin_id
+
+    s3_origin_config {
+      origin_access_identity = aws_cloudfront_origin_access_identity.origin_access_identity.cloudfront_access_identity_path
+    }
+  }
+
+  enabled             = true
+  is_ipv6_enabled     = true
+
+  aliases = [var.domain]
```

Comment:
Na zacatku souboru bych ty variables nadefinoval, aby bylo hned videt, ze je potreba je pridat do ENV (ale to je asi muj bias, prichazim z type-safe sveta)

```
variable domain {
 type = string
}
```